data_model/README.md

```diff
@@ -2,30 +2,115 @@
 orphan: true
 ---
 
-## Contents
+# Data model XML files
 
 This folder contains a machine-readable representation of matter clusters.
 
-The XML files inside `clusters` are generated by a `scraper` script out of the
-original specification `AsciiDoc` files.
+These files are currently used for certification testing and documentation. The
+data model files represent the official specification data model at a certain
+revision. The files in these directories are maintained by the data model tiger
+team (DMTT) and are updated when new official specifications or specification
+ballots are released.
 
-## How to update
+The data model files in this directory are not used with zap or for SDK codegen
+and should not be updated manually by cluster or device type implementers.
 
-The matter specification is not currently public. As such, as script exists to
-update the spec XML files, however this is not done automatically.
+## Updating the data model files
 
-You will require access to the following tools locally:
+### Updating current spec revision directories
 
--   `scraper`. A binary copy generally available
-    [here](https://github.com/csa-data-model/projects/tree/main/DM-Editor/bin/scrape)
--   Specification repository checkout from
-    https://github.com/CHIP-Specifications/connectedhomeip-spec
+If the files exist in the data_model directory already and you are trying to
+update either the revision or the alchemy release, use
+scripts/spec_xml/generate_spec_xml.py. More information on the use of the script
+can be found by using the --help flag for that script.
 
-Example usage:
+Perform the following steps
 
-```sh
-./scripts/spec_xml/generate_spec_xml.py     \
-   --scraper ~/Downloads/scrape-adoc-linux  \
-   --spec-root ~/work/connectedhomeip-spec  \
-   --output-dir data_model
-```
+-   Check out the specification repo at the desired sha/tag/branch
+    (https://github.com/CHIP-Specifications/connectedhomeip-spec)
+-   Determine the desired include level (current ballot includes, no in-progress
+    includes or all in-progress includes). If you are generating files using
+    "Current", be sure to double-check the include list inside the script
+    against the ballot email.
+-   Run the script and check in all the file changes
+-   You can use the scripts/spec_xml/spec_revision_diff_summary.py script to
+    create a summary with the differences between the data model files for two
+    different directories
+-   Be sure to double-check the summary and generated ID files to ensure the
+    provisional markings are as expected
+-   If you are updating both the spec SHA and the scraper version, it is best to
+    do this in two separate commits so reviewers can see which changes come from
+    the spec and which changes come from the scraper additions
+-   Before pushing the PR, run build_python.sh, activate the venv and run all
```

Comment:
commants for this would be helpful as well. these will for sure be copied and pasted

Reply:
added.